content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,249 @@
+---
+title: "Sourcegraph 4.0"
+publishDate: 2022-09-27T10:00-07:00
+description: Announcing Sourcegraph 4.0
+tags: [blog, release]
+slug: "release/4.0"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/4.0/release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/4.0/release.png
+changelogItems:
+---
+
+WIP INTRO: 
+
+We've spent 9 years building the underlying foundation of code search to be fast, secure, and scalable across ~10^11 lines of code for the world's most demanding software teams. Now, we’re hard at work making Sourcegraph even smarter.
+
+Sourcegraph 4.0 includes over a dozen updates and new features, and is the first step toward Sourcegraph becoming a [code intelligence platform](blog/code-search-to-code-intelligence).
```

Comment:
```suggestion
Sourcegraph 4.0 includes over a dozen updates and new features, and is the first step toward Sourcegraph becoming a [code intelligence platform](blog/code-search-to-code-intelligence).

A code intelligence platform indexes and understands all of your organization’s code, so instead of just *matching lines in code*, you can answer questions and learn *about* code. 
```